app/controllers/authors_controller.rb

```diff
@@ -65,6 +65,8 @@ def show
       if all_posts.count > POST_LIMIT && all_posts.last.created_at < @posts.last.created_at
         @posts.last.created_at.to_i
       end
+    @should_show_condensed_cover = @display_author.cover_style == "condensed"
```

Comment:
"condensed" and other options should be constants somewhere and not magic strings whose values are not predefined.

Reply:
agree, fixed! @mobitar